src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+.page {
+  --active-star: url(/src/images/star-active.svg);
+}
+
+.page__body {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :first-child {
+  background-image: var(--active-star);
+}
+
+.stars--2 :nth-child(-n + 2) {
```

Comment:
Similar to the previous comment, `.stars--2 :nth-child(-n + 2)` has a space before `:nth-child`, which targets the first two children of any element inside `.stars--2`. Remove the space to target the first two `.stars__star` elements directly inside `.stars--2`: `.stars--2:nth-child(-n + 2)`.